### PR TITLE
Module config: mention possible need for proxy setting

### PR DESCRIPTION
**Pull request by deining**
This PR adds the `proxy` attribute to the module configuration, thus alleviating the use of a proxy.

### CODE REVIEW

**Comment by deining** [comment]
Added a note on the module top level settings to `README.md`, as disussed [here](https://github.com/google/docsy/pull/994#issuecomment-1126297124).

File: README.md [hunk 1]
```diff
@@ -37,7 +37,10 @@ A simple way to get started is to use this project as a template, which gives yo
 git clone --depth 1 https://github.com/me/example.git
 ```
 
-You can now edit your own versions of the site’s source files.
+Depending on your environment you may need to tweak the [module top level settings](https://github.com/google/docsy-example/blob/1c7f7e300c90cd690ca5be66b43fe58713bb21c9/config.toml#L221-L228) inside your `config.toml` slightly, for example by adding a proxy to use when downloading remote modules.
```
Comment (chalin) [inline README.md:40]:
```suggestion
Depending on your environment, you may need to adjust the top-level `module` settings in your project's Hugo config file`, for example, by adding a proxy to use when downloading remote modules.
```

File: hugo-disabled.toml
Comment (chalin) [inline hugo-disabled.toml:1]: Hmm, maybe we should avoid making changes to this "disabled" config file, though I realize that they were made before the config file was renamed. You call.

File: README.md (hunk 1 above)
Comment (deining) [reply README.md]: Changed as proposed.

File: hugo-disabled.toml
Comment (deining) [reply hugo-disabled.toml:1]: I would opt for leaving these changes in place.

**Review by chalin — APPROVED** [review]

**Pull request merged**